tests/test_locales.py

```diff
@@ -53,6 +53,22 @@ def test_get_locale(self, mocker):
 
         assert result == mock_locale
 
+    def test_get_locale_by_class_name(self, mocker):
```

Comment:
nice, thanks for this! It didn't even cross my mind to put up a test for this when I was doing it.

Reply:
Heh, I just noticed that running the tests locally failed technically on coverage < 100% and this new function was easy enough to test. :wink: